.scripts/util.sh

```diff
@@ -126,25 +129,26 @@ copy_ct60_modules() {
 
 	# we don't need plain st/falcon version
 	rm "$MCHDIR/rtl8012.xix"
-	cp "$SRC/sys/sockets/xif/rtl8012ct60.xif" "$MCHDIR/rtl8012.xix"
-	cp "$SRC/sys/sockets/xif/rtl8139.xif" "$MCHDIR/rtl8139.xix"
-	cp "$SRC/sys/sockets/xif/ethernat/ethernat.xif" "$MCHDIR/ethernat.xix"
-	cp "$SRC/sys/sockets/xif/svethlana/svethlan.xif" "$MCHDIR/svethlan.xix"
+	cp "$SRC/sys/sockets/xif/rtl8012_060.xif" "$MCHDIR/rtl8012.xix"
+	cp "$SRC/sys/sockets/xif/rtl8139_060.xif" "$MCHDIR/rtl8139.xix"
+	cp "$SRC/sys/sockets/xif/ethernat/.compile_060/ethernat.xif" "$MCHDIR/ethernat.xix"
+	cp "$SRC/sys/sockets/xif/svethlana/.compile_060/svethlan.xif" "$MCHDIR/svethlan.xix"
 }
 copy_firebee_modules() {
 	local MCHDIR="$1/firebee"
 	mkdir -p "$MCHDIR"
-	cp "$SRC/sys/sockets/xif/fec/fec.xif" "$MCHDIR/fec.xif"
+	cp "$SRC/sys/sockets/xif/fec/.compile_v4e/fec.xif" "$MCHDIR/fec.xif"
 }
 copy_aranym_modules() {
 	local MCHDIR="$1/aranym"
 	mkdir -p "$MCHDIR"
-	cp "$SRC/sys/sockets/xif/nfeth/nfeth.xif" "$MCHDIR"
-	cp "$SRC/sys/xdd/nfstderr/nfstderr.xdd" "$MCHDIR"
-	cp "$SRC/sys/xdd/nfexec/nfexec.xdd" "$MCHDIR"
+	cp "$SRC/sys/sockets/xif/nfeth/.compile_040/nfeth.xif" "$MCHDIR"
+	cp "$SRC/sys/xdd/nfstderr/.compile_040/nfstderr.xdd" "$MCHDIR"
+	cp "$SRC/sys/xdd/nfexec/.compile_040/nfexec.xdd" "$MCHDIR"
+	cp "$SRC/sys/xdd/nfcdrom/.compile_040/nfcdrom.xdd" "$MCHDIR/nfcdrom.xdx"
 	cp "$SRC/sys/xfs/aranym/aranym.xfs" "$MCHDIR"
 	# not really needed
-	cp "$SRC/sys/xfs/hostfs/hostfs.xfs" "$MCHDIR/hostfs.xfx"
+	cp "$SRC/sys/xfs/hostfs/.compile_000/hostfs.xfs" "$MCHDIR/hostfs.xfx"
```

Comment:
Why _000 and not _040?

Reply:
Because the standalone module is compiled for 000. For the aranym kernel, that filesystem is compiled into the kernel anyway. The standalone module would only be useful for other CPU targets, but then you can of course not boot from hostfs (but theoretically it could still be used to access other drives)

Reply:
OK, fair point. However then it makes no sense to put it into Aranym's MCHDIR because that implies it is usable only for the aranym kernel.